test/integration/crud/find_and_modify.test.ts

```diff
@@ -0,0 +1,393 @@
+import { expect } from 'chai';
+
+import { type CommandStartedEvent, MongoServerError, ObjectId } from '../../mongodb';
+import { setupDatabase } from '../shared';
```

Comment:
Do we have any tests for `includeResultMetadata = false`?  I see tests for the default behavior, but nothing with `includeResultMetadata` disabled.

Reply:
Added tests with the option set to false.